IssueTracker/IssueTracker.FileSystem/Repository/BolbData.cs

```diff
@@ -41,8 +43,11 @@ public async Task UploadFileAsync(Models.File file)
     {
         if (file == null)
             throw new ArgumentException("You don't have files!");
-
-        await ContainerClient.CreateIfNotExistsAsync();
+        if (file.BlobName == String.Empty || file.BlobName == null)
+            throw new ArgumentException("Invalid BlobName");
+        if (file.Content == null)
+            throw new ArgumentException("Invalid Content");
+        //await ContainerClient.CreateIfNotExistsAsync();
```

Comment:
keep the code clean

Reply:
<img width="421" alt="image" src="https://user-images.githubusercontent.com/74471664/190163383-e822ff63-afe4-43f8-8925-f29ddcb3b99f.png">